### PR TITLE
test: add funcional test to scrollview component

### PR DESCRIPTION
**Pull request by jenifersantoszup**
### Related Issues
#1159 

### Description and Example

Functional tests were created for the scrollview component to meet the scrollview scenario within scrollview.

### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.

[PR Guide]: https://github.com/ZupIT/beagle/blob/master/doc/contributing/pull_requests.md

### CODE REVIEW

File: iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift [hunk 1]
```diff
@@ -45,9 +45,15 @@ class ScrollViewSteps: CucumberStepsDefinition {
             self.application.staticTexts[text].tap()
         }
         
+        //Scenario 2
+        When(#"^I press on text "([^\"]*)" of scrollview$"#) { args, _ -> Void in
+            let text = args![0]
```
Comment (botbeaglezup) [inline iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift:50]: <table data-meta="generated_by_ios">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/41e677ca2dbea5e14956c3cf51c0a8f9d522bb7f/iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift#L50"></span>Force unwrapping should be avoided.<br />
<code>force_unwrapping</code> <code>ScrollViewSteps.swift:50</code></td>
    </tr>
  </tbody>
</table>

File: iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift [hunk 2]
```diff
@@ -45,9 +45,15 @@ class ScrollViewSteps: CucumberStepsDefinition {
...
+            self.application.staticTexts[text].tap()
+        }
+        
         // MARK: - Then
         
-        // Scenario 1
+        // Scenario 1, 2
         Then(#"^the current text "([^\"]*)" should be replaced for a large text and It should scroll to the "([^\"]*)" button for tapping it$"#) { args, _ -> Void in
             let text = args![0]
```
Comment (botbeaglezup) [inline iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift:58]: <table data-meta="generated_by_ios">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/41e677ca2dbea5e14956c3cf51c0a8f9d522bb7f/iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift#L58"></span>Force unwrapping should be avoided.<br />
<code>force_unwrapping</code> <code>ScrollViewSteps.swift:58</code></td>
    </tr>
  </tbody>
</table>

File: iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift [hunk 3]
```diff
@@ -45,9 +45,15 @@ class ScrollViewSteps: CucumberStepsDefinition {
...
             let buttonTitle = args![1]
```
Comment (botbeaglezup) [inline iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift:59]: <table data-meta="generated_by_ios">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/41e677ca2dbea5e14956c3cf51c0a8f9d522bb7f/iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift#L59"></span>Force unwrapping should be avoided.<br />
<code>force_unwrapping</code> <code>ScrollViewSteps.swift:59</code></td>
    </tr>
  </tbody>
</table>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          4 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/41e677ca2dbea5e14956c3cf51c0a8f9d522bb7f/iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift#L21">iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift#L21</a> - Implicitly unwrapped optionals should be avoided when possible.<br />
<code>implicitly_unwrapped_optional</code> <code>ScrollViewSteps.swift:21</code></td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/41e677ca2dbea5e14956c3cf51c0a8f9d522bb7f/iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift#L42">iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift#L42</a> - Force unwrapping should be avoided.<br />
<code>force_unwrapping</code> <code>ScrollViewSteps.swift:42</code></td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/41e677ca2dbea5e14956c3cf51c0a8f9d522bb7f/iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift#L43">iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift#L43</a> - Force unwrapping should be avoided.<br />
<code>force_unwrapping</code> <code>ScrollViewSteps.swift:43</code></td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/41e677ca2dbea5e14956c3cf51c0a8f9d522bb7f/iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift#L81">iOS/Tests/AutomatedTestsUITests/Steps/ScrollViewSteps.swift#L81</a> - Prefer <code>private</code> over <code>fileprivate</code> declarations.<br />
<code>private_over_fileprivate</code> <code>ScrollViewSteps.swift:81</code></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_ios">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by lucasfonsecazup — DISMISSED** [review]

**Review by carlossteinzup — DISMISSED** [review]

File: android/automated-tests/src/androidTest/assets/features/ScrollView.feature [hunk 4]
```diff
@@ -30,27 +30,24 @@ Feature: ScrollView Component Validation
...
-            | Click to see the new text in horizontal |
+            | textScrollHorizontal                    | buttonScrollVertical |
+            | Click to see the new text in horizontal | horizontal scroll    |
 
     Scenario Outline: ScrollView 03 - scrollView component should be render the correctly texts in horizontal even if the screen is rotated
         When I press on text to be scrolled and rotated <textScrollViewRotate>
-        Then the text horizontal of scrollview rotate should change
-        And the scrollview rotate should perform horizontally <textScrollViewRotate>
-        And even if the screen is rotated the scrollview must be perform horizontally <textScrollViewRotate>
+        Then the text <textScrollViewRotate> of horizontal of scrollview rotate should change
```
Comment (hthemir) [inline android/automated-tests/src/androidTest/assets/features/ScrollView.feature:42]: Should this line remove the first "of", as in "Then the text <textScrollViewRotate> horizontal of scrollview rotate should change" ?

**Review by hthemir — DISMISSED** [review]

**Review by hthemir — APPROVED** [review]

**Review by carlossteinzup — APPROVED** [review]

**Review by lucasfonsecazup — APPROVED** [review]

**Pull request merged**